neuralmonkey/readers/image_reader.py

```diff
@@ -0,0 +1,73 @@
+from typing import Callable, List, Optional
+import os
+import numpy as np
+from scipy import ndimage
```

Comment:
máme scipy v requirements? máme to tam přidat? máme to jen zmínit v dokumentaci? (podobně jako bychom to měli udělat u nltk bleu evaluátora, až/jestli se přimerǧne z dekoderiho refaktoru)